src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx

```diff
@@ -72,6 +72,17 @@ export default function NewViabilityTestModal(props: NewViabilityTestModalProps)
     onClose();
   };
 
+  const getSubstratesAccordingToType = (type?: string) => {
+    if (type) {
+      if (type === 'Lab') {
+        return LAB_SUBSTRATES;
+      } else {
+        return NURSERY_SUBSTRATES;
+      }
+    }
+    return [];
```

Comment:
suggestion (or also switch statement) to avoid nested-ifs:
```
if (type === 'Lab') {
  return LAB_SUBSTRATES;
} else if (type === 'Nursery') {
  return NURSERY_SUBSTRATES;
} else 
  return [];
}